tests/bsim/bluetooth/host/adv/chain/prj_extra_chain.conf

```diff
@@ -0,0 +1,41 @@
+CONFIG_BT=y
+CONFIG_BT_BROADCASTER=y
+CONFIG_BT_OBSERVER=y
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_EXT_ADV_MAX_ADV_SET=2
+#
+# The extra long device name requires chaining an additional PDU in this test-case
+CONFIG_BT_DEVICE_NAME="Broadcaster Multiple with an extra long name 12345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890"
+
+# Enable Advertising Data chaining in Zephyr Bluetooth LE Controller
+CONFIG_BT_CTLR_ADVANCED_FEATURES=y
+CONFIG_BT_CTLR_ADV_DATA_CHAIN=y
+
+# Zephyr Bluetooth LE Controller will need to use chain PDUs when AD data
+# length > 191 bytes
+# - 31 bytes will use 22 bytes for the default name in this sample plus 9 bytes
+#   for manufacturer data
+# - 191 bytes will use 22 bytes for the default name in this sample plus 169
+#   bytes for manufacturer data
+# - 277 bytes will use 22 bytes for the default name in this sample plus 255
+#   bytes for manufacturer data
+CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=1650
+
+# Increase Advertising PDU buffers to number of advertising sets times the
+# number of chain PDUs per advertising set when using Zephyr Bluetooth LE
+# Controller
+CONFIG_BT_CTLR_ADV_DATA_BUF_MAX=2
+
+# Maximum Extended Scanning buffer size
+CONFIG_BT_EXT_SCAN_BUF_SIZE=1650
+
+# Set maximum scan data length for Extended Scanning in Bluetooth LE Controller
+CONFIG_BT_CTLR_SCAN_DATA_LEN_MAX=1650
+
+# Zephyr Bluetooth LE Controller needs 16 event buffers to generate Extended
+# Advertising Report for receiving the complete 1650 bytes of data
+CONFIG_BT_BUF_EVT_RX_COUNT=16
```

Comment:
Why 16? Each report can hold up to 229 octets, so for 1650 octets at most 8 events are required

Reply:
Hm, this was copied from prj.conf

Reply:
8 + 8, is used as Zephyr Controller does not combine all the 1650 bytes before fragmenting them into 8 HCI reports, if a PDU has 255 bytes, it will generate 2 HCI reports.

Reply:
I see. Can we update the comment to state this? It just says that it needs 16, but not _why_

Reply:
Updated the comment to explain why